### ISSUE TITLE
Support Dark Mode?

### ISSUE THREAD

**Issue by johncoxon** [angstsmurf/spatterlight#72]
It would be very nice to be able to select a theme to use in Dark Mode and a theme to use in Light Mode.

**Comment by angstsmurf** [comment]
That would be very nice! The hard part is really to come up with a simple, intuitive, non-confusing UI for it. I have some ideas.

**Comment by angstsmurf** [comment]
Here is a very experimental version. It would be great if you could give it a whirl and let me know what you think.

There are two ways to designate themes for light or dark mode: checkboxes in the Details tab of the preferences window, and new menu items in the Themes tab menu (Use for Light Mode and Use for Dark Mode).

I am particularly curious about how it works with game-specific themes, i.e. when the "Changes apply to all games in library" checkbox on the Themes tab is off. This was always kind of confusing and most likely not used by anyone. Does it behave as you would expect when you switch off "Use for X Mode" and it goes back to the old behaviour? 

[Spatterlight.zip](https://github.com/angstsmurf/spatterlight/files/11476142/Spatterlight.zip)

**Comment by Nephar** [comment]
Seems to work OK in Mojave.

When `Changes apply to all games in library` is on, all games switch to whatever I have chosen for `Use for X Mode`. And if the system is in dark mode and I have a dark mode theme set, then I cannot apply a different theme from the game menu(grayed), which is as it should be.

When `Changes apply to all games in library` is off, `Use for X Mode` forces all games to use same theme in that mode, but if no theme is set for the other mode, all games switch back to their own preferred theme when I switch to that mode.

In summary, I found no inconsistencies in the behaviour.

**Comment by angstsmurf** [comment]
Thanks! That is good to hear.

Let me know if any bugs turn up or if there is anything that should be improved.

**Comment by johncoxon** [comment]
This is perfect; I couldn't find anything wrong but I only have one .z5 file to test with. I'll post here if I do come across any inconsistencies. Thank you so much.

**Comment by angstsmurf** [comment]
This is now included in [release 1.1](https://github.com/angstsmurf/spatterlight/releases/tag/v1.1).
